scripts/deploy-apm.js

```diff
@@ -76,12 +77,18 @@ module.exports = async (
   } else {
     log('Deploying DAOFactory with EVMScripts...')
     daoFactory = (await deployDaoFactory(null, { artifacts, withEvmScriptRegistryFactory: true, verbose: false })).daoFactory
-    log('Deployed DAOFactory:', daoFactory.address)
   }
 
   log('Deploying APMRegistryFactory...')
-  const apmFactory = await APMRegistryFactory.new(daoFactory.address, ...apmBases, ensAddress, '0x00')
-  log('Deployed APMRegistryFactory:', apmFactory.address)
+  const apmFactory = await APMRegistryFactory.new(
+    daoFactory.address,
+    apmRegistryBase.address,
+    apmRepoBase.address,
+    ensSubdomainRegistrarBase.address,
+    ensAddress,
+    '0x00'
+  )
+  await logDeploy(apmFactory, { verbose })
 
   log(`Assigning ENS name (${labelName}.${tldName}) to factory...`)
   try {
```

Comment:
Tried `truffle exec --network rpc scripts/deploy-apm.js` by itself and hit the ENS error here.

Maybe we should also set `aragonpm.eth` if we deploy our own instance of ENS in this script?

Reply:
It blows up if you don't pass the first account as the `OWNER` env variable, fixing it.

Reply:
Added this in 04813a1:

```
...
OWNER env variable not found, setting APM owner to the provider\'s first account
Owner: 0xb4124ceb3451635dacedd11767f004d8a28c6ee7
...
```

If you run `truffle exec --network rpc scripts/deploy-apm.js` now it should just work.